openshift/template.yaml

```diff
@@ -27,7 +27,22 @@ objects:
             value: "${GUNICORN_CMD_ARGS}"
           image: "${DOCKER_REGISTRY}/${DOCKER_IMAGE}:${IMAGE_TAG}"
           name: f8a-3scale-admin-gateway
-          ports:
+          - name: THREE_SCALE_API_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                name: 3scale
+                key: three_scale_api_access_key
+          - name: THREE_SCALE_API_HOST_URL
```

Comment:
This is not a secret, we can get the value from config map:

```
          - name: THREE_SCALE_API_ACCESS_KEY
            valueFrom:
              configMapKeyRef:
                name: bayesian-config
                key: three-scale-api-host-url
```

Reply:
This change requires https://github.com/fabric8-analytics/fabric8-analytics-common/pull/360.

Reply:
@msrb, I have removed it. :)